src/System/Analytics/AnalyticsContext.tsx

```diff
@@ -63,9 +66,16 @@ export const AnalyticsContextProvider: FC<AnalyticsContextProviderProps> = ({
   ])
 
   const contextPageOwnerType = useMemo(() => {
-    return (path
+    const ownerType = (path
       ? pathToOwnerType(path) || formatType(_type) // Fallback for unsupported routes
       : "Unknown") as PageOwnerType
+
+    if (!Object.values(OwnerType).includes(ownerType) && SHOW_ANALYTICS_CALLS) {
+      console.warn(
+        `OwnerType "${ownerType}" is not part of @artsy/cohesion's schema.`
+      )
+    }
```

Comment:
It is a shame we can't test `PageOwnerType` rather than `OwnerType` - I think this is because [it is a union](https://github.com/artsy/cohesion/blob/main/src/Schema/Values/OwnerType.ts#L172) ('x' | 'y' | 'z') rather than an enum, and enums [act like objects at runtime](https://www.typescriptlang.org/docs/handbook/enums.html) so can interact with normal javascript code.

I wish there were a similar typescript utility to `Pick<>` for these.

Reply:
thanks for walking through and helping me understand why this doesn't work btw!